cli/cli.go

```diff
@@ -0,0 +1,77 @@
+package cli
+
+import (
+	. "fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/codegangsta/cli"
+	"github.com/ugorji/go/codec"
+
+	"polydawn.net/repeatr/def"
+	"polydawn.net/repeatr/executor"
+	"polydawn.net/repeatr/executor/null"
+)
+
+func GetApp() *cli.App {
+	app := cli.NewApp()
+
+	app.Name = "repeatr"
+	app.Usage = "Run it. Run it again."
+	app.Version = "0.0.1"
+
+	app.Commands = []cli.Command{
+		{
+			Name:   "run",
+			Usage:  "Run a forumla",
+			Action: Run,
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "executor, e",
+					Value: "null",
+					Usage: "Which executor to use",
+				},
+				cli.StringFlag{
+					Name:  "input, i",
```

Comment:
We're already overloading the word 'input' in way too many places, what do you think of calling this 'formula' in the flag instead?